src/jdk.attach/linux/native/libattach/VirtualMachineImpl.c

```diff
@@ -111,6 +111,46 @@ JNIEXPORT void JNICALL Java_sun_tools_attach_VirtualMachineImpl_connect
         }
     }
 }
+/**
+ * Check /proc/$pid/stat to determine SIGQUIT is caught by pid.
+ * Return 1 if the SIGQUIT is set in SigCgt; 0 if it is not.
+ * Return -1 when it runs into any error.
+ */
+static int check_sigquit_caught(jint pid) {
```

Comment:
can't this just be a bool function? Even if int we don't need a ternary return value as only zero is of interest.